packages/autocomplete-core/src/getPropGetters.ts

```diff
@@ -31,46 +33,53 @@ export function getPropGetters<
   const getEnvironmentProps: GetEnvironmentProps = (providedProps) => {
     const { inputElement, formElement, panelElement, ...rest } = providedProps;
 
-    return {
-      // On touch devices, we do not rely on the native `blur` event of the
-      // input to close the panel, but rather on a custom `touchstart` event
-      // outside of the autocomplete elements.
-      // This ensures a working experience on mobile because we blur the input
-      // on touch devices when the user starts scrolling (`touchmove`).
+    function onMouseDownOrTouchStart(event: MouseEvent | TouchEvent) {
+      // The `onTouchStart`/`onMouseDown` events shouldn't trigger the `blur`
+      // handler when it's not an interaction with Autocomplete.
+      // We detect it with the following heuristics:
+      // - the panel is closed AND there are no pending requests
+      //   (no interaction with the autocomplete, no future state updates)
+      // - OR the touched target is the input element (should open the panel)
+      const isAutocompleteInteraction =
+        store.getState().isOpen || !store.pendingRequests.isEmpty();
+
+      if (!isAutocompleteInteraction || event.target === inputElement) {
+        return;
+      }
+
       // @TODO: support cases where there are multiple Autocomplete instances.
       // Right now, a second instance makes this computation return false.
```

Comment:
I can't see how this would be an issue with multiple instances. Aren't `formElement` / `panelElement` unique to each instance of Autocomplete?